### DISCUSSION TITLE
how to hide already selected item in useFieldArray from select menu options.

### DISCUSSION THREAD

**Discussion by Sbabureddy** [react-hook-form/react-hook-form#11369] [General] [answered]
Hi Team, 

I am working on `useFieldArray`. if user selects a country from the list and try to append another country. in the next country dropdown list the country already selected shouldn't be visible.

example: if I select `India` first time then try to append another country, in the next dropdown `India` shouldn't be visibile.

https://codesandbox.io/p/sandbox/react-hook-form-usefieldarray-rules-forked-wmcmlk

**Comment by chanthinh** [comment]
@Sbabureddy, your codesandbox is working exactly what you want. What is the issue?

**Reply by Moshyfawn** [reply]
I suppose they've found the solution, because it's changed since I first changed it, and it didn't work as described.

**Comment by Sbabureddy** [comment, marked as answer]
@Moshyfawn  @chanthinh Thanks for the reply. is there anyway to send both value and label  of html select option in react hook form.

In the above codesandbox like: {countryName: 'India', countryCode: 'IND'} for country selection.
